notebooks/en/_toctree.yml

```diff
@@ -0,0 +1,12 @@
+- title: Open-Source AI Cookbook
+  sections:
+  - local: FAISS_with_HF_datasets_and_CLIP
+    title: Embedding multimodal data for similarity search using 🤗 transformers, 🤗 datasets and FAISS
```

Comment:
Small nit: but I think titles should be usually shorter. Otherwise, they would look too long on the navigation side bar